src/gui/folderman.cpp

```diff
@@ -145,8 +145,8 @@ FolderMan::FolderMan(QObject *parent)
 
     _socketApi.reset(new SocketApi);
 
-    // Set the remote poll interval fixed to 10 seconds.
-    // That does not mean that it polls every 10 seconds, but it checks every 10 secs
+    // Set the remote poll interval fixed to 1 second (1000 milliseconds)
```

Comment:
That pretty much looks like a bug instead, 1s polling sounds ridiculous.

Reply:
Looking of the change that introduces that, yes that should be 10s.
Could you instead of fixing the doc, replace the `1000` with `10s`

Reply:
@TheOneRing you mean replace `1000` with `10000` ?

IMO the side-effect will be that if someone sets the server to advertise a `pollinterval` of 8 seconds, for example, then actually that will not be effective. This code here will only wake up every 10 seconds to see if the `pollinterval` has expired.

Maybe set it to `5000` (5 seconds)? And then the client will respect accurately a server `pollinterval` that is a multiple of 5000. (And respect less accurately a `pollinterval` like 18000)

Reply:
So we have a poll interval per sync folder pair, this can be multiple and we again have multiple accounts.
The poll interval here only calls a function that iterates over the sync folder pairs and looks for those in need of a fresh poll.
The whole code should be reworked as it offers lots of limitations, and we plan to have parallel syncs in the future.

I meant `_etagPollTimer.setInterval(10s);` a newer version of C++ now allows us to use seconds as type.

PS: I think a poll interval below the random value of 30s is pointless.

Reply:
done